java/vector/pom.xml

```diff
@@ -74,6 +74,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I'm a little hesitant to take a direct dependency on any lz4 library.  Is there away that this can be done optionally (similar to how the netty dependency for memory has been isolated?)

Reply:
@emkornfield Sounds reasonable. I will try to revise the PR accordingly. Thanks for your good suggestion.